canonical/lxd-ui
# Configurable backend ip for dev mode WD-3856

## Done

- read the backend ip for the dev mode from `.env` file and allow overriding it in a `.env.local` file, also respecting commenting out of the ip
- updated last step of cluster documentation at https://github.com/canonical/lxd-ui/wiki/Setup-local-LXD-cluster

Fixes [WD-3856](https://warthogs.atlassian.net/browse/WD-3856)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - ensure dev setup still works
    - ensure overriding in .env.local works

[WD-3856]: https://warthogs.atlassian.net/browse/WD-3856?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
